src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+    private static final StorageDao STORAGE_DAO = new StorageDaoImpl();
+    private static final User DEFAULT_USER = new User();
+    private static final String CORRECT_LOGIN = "Johnson";
+    private static final String CHECK_LOGIN = "Lara";
+    private static final int CORRECT_AGE = 20;
+    private static final int YOUNG_AGE = 10;
```

Comment:
for testing it is better to take marginally unacceptable input data, in this case 17 years (check  another  test data)